### ISSUE TITLE
Component props height and width not working

### ISSUE THREAD

**Issue by jgutierrez11** [hannigand/react-native-ui-stepper#6]
Hi, I am trying to implement this component in my screen but it doesn't apply the height and width I'm passing, so the component takes a large portion of the screen....

Even when I put ridiculous small values, it shows a pretty big stepper.
Here's my code:

```
<UIStepper
              imageWidth={20}
              imageHeight={10}
              width={94}
              height={29}
              displayValue={true}
              initialValue={0}
              steps={1}
              borderColor={'transparent'}
              tintColor={'#444'}
              minimumValue={0}
              textColor={'#38B4B7'}
              incrementImage={require('../../utils/images/common/stepper_plus.png')}
              decrementImage={require('../../utils/images/common/stepper_minus.png')}
              onValueChange={(value) => {this._onStepperValueChanged(value)}}
            />
```

**Comment by hannigand** [comment]
Hey @jgutierrez11 - thanks for submitting the issue.

I will take a look at this later today and push out a potential fix. 

Sorry for any inconveniences that this issue has caused.

**Comment by hannigand** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Fix measurement props](https://trello.com/c/DNZgPZMi/1-fix-measurement-props)

**Comment by hannigand** [comment]
@jgutierrez11 a fix has been pushed to version 1.2.1.
